modules/nw-sriov-about-qinq.adoc

```diff
@@ -11,7 +11,13 @@ In traditional VLAN setups, frames typically contain a single VLAN tag, such as
 
 QinQ facilitates the creation of nested VLANs by using double VLAN tagging, enabling finer segmentation and isolation of traffic within a network environment. This approach is particularly valuable in service provider networks where you need to deliver VLAN-based services to multiple customers over a common infrastructure, while ensuring separation and isolation of traffic.
 
-image::693_OpenShift_QinQ_SR-IOV_CNI_0624.png[QinQ]
+The following diagram illustrates how {product-title} can leverage SR-IOV and QinQ to achieve advanced network segmentation and isolation for containerized workloads.
+
+The diagram shows how QinQ (double VLAN tagging) works in a worker node with SR-IOV support. It shows a pod, `Pod A` with a network interface, `ext0` connected to an SR-IOV NIC that provides multiple Virtual Functions (VFs). Network traffic from `Pod A` is tagged with a customer VLAN (C-tag) within the pod and an additional service VLAN (S-tag) as it exits through the VFs. 
```

Comment:
Let's replace "It shows a pod, ......" with:

The SRIOV VF located in the pod namespace (ext0) is configured (by the sriov cni) with a vlan ID and vlan protocol. This corresponds to the s-tag. Within the pod, a vlan subinterface (i.e. created by the vlan cni) that has as master the interface ext0, add the internal vlan ID with procotol 802.1Q. This corresponds to the c-tag.

Something like that.

Reply:
@mlguerrero12 I updated hopefully OK for LGTM